hrm_class.py

```diff
@@ -0,0 +1,231 @@
+import numpy
+import pandas
+
+class hrm_data:
+    """This is a HRM class
+
+       __init__ finds the peaks and sets the initial parameters
+       instantHr finds the instantaneous hr at a given time
+       averageHr finds the average hr over a period of time
+       anomalyHr determines if there is brady/tachy and where they occurred
+
+       Attributes:
+            times (list): list of times that a peak occurs
+            inst   (int): the last instantaneous hr computed
+            avg    (int): the last average hr computed
+            ano   (list): list of times the anomalies occurred
+            bradyTimes
+            tachyTimes
+
+    """
+
+    def __init__(
+            self,
+            filename,
+            begin_time=0,
+            end_time=10,
+            bradyT=5,
```

Comment:
as with elsewhere in the code, a reminder to use snake_case and generally be consistent with naming :)